src/runtime/TabsController.ts

```diff
@@ -51,6 +53,43 @@ export class TabsController {
                 this._selectTab(tab, target);
             }
         });
+        this._document.addEventListener('keydown', (event) => {
+            let direction: TabSwitchDirection | null = null;
+            switch (event.key) {
+                case 'ArrowLeft': {
+                    direction = 'left';
+                    break;
+                }
+                case 'ArrowRight': {
+                    direction = 'right';
+                    break;
+                }
+            }
+            if (!direction) {
+                return;
+            }
+
+            const target = getEventTarget(event) as HTMLElement;
```

Comment:
How will it work for tabs inside tabs?

Reply:
Good question. Most probably I need to add an extra handling for this cases. Let me check

Reply:
With nested tabs there is 2 problems:  1) on first YFM transformation plugin render as we can wee [here](https://github.com/diplodoc-platform/tabs-extension/blob/09ff30b57cd56f19361d61b9ca109142bc9f2c95/src/plugin/transform.ts#L188) the first tab is always open. In case of nested tabs: first nested tab will be opened, thus, its content can be already visible, but it is not correct behaviour if nested tabs should not be visible. And 2) for roving tabindex to be working we need to have first nested tab opened already. If one presses the parent tab, all nested tabs will be closed, thus, tabindex=-1 on all of them. For this reason they are not accessible via keyboard. I will create issues related to this problems

Reply:
UPD: the bug is already fixed. I was developing on version 2.0.15. After rebase it is working correctly now

Reply:
For nested tabs I have created additional unit tests. Everything seems working correctly